scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/controller/ExecutionsController.java

```diff
@@ -73,18 +73,10 @@ public JobsController getJobsController() {
         return jobsController;
     }
 
-    public void setJobsController(JobsController jobsController) {
```

Comment:
Does TaskCentricController replaces JobsController as well? So there is no difference Jobs/Tasks anymore?